content/metadata.yaml

```diff
@@ -18,52 +18,60 @@ author_info:
     affiliations:
       - Molecular Biosciences and Bioengineering Graduate Program, University of Hawaii at Manoa, Honolulu, HI
     symbol_str: "☯"
+    v1: true
```

Comment:
The convention you propose is that all v1 authors must make new contributions to be designated as v2 authors as well, right?  Then at a later date we would decide how to merge v1 and v2 author lists?

Reply:
I was thinking that we'd probably make the "v1" authors into a "boxed" author, but I don't have strong feelings on this. That approach most fits with current convention, but perhaps there is a better way.

Reply:
So we'd add a consortium author like "Deep Review 1.0 Authors" (with a fancier name)?  That makes sense to me.  We can open an issue to discuss with the 1.0 authors once it advances that far.

Reply:
Yea - this is the route I was imagining going.